packages/extension-ui/src/Popup/Derive/SelectParent.tsx

```diff
@@ -143,7 +150,8 @@ function SelectParent ({ className, isLocked, onDerivationConfirmed, parentAddre
                 parentAddress={parentAddress}
                 parentPassword={parentPassword}
               />
-              {!suriPath && <div className='error'>{t('Incorrect derivation path')}</div>}
+              {!suriPath && <Warning isBelowInput
+                isDanger>{t('Incorrect derivation path')}</Warning>}
```

Comment:
```suggestion
              {!suriPath && (
                <Warning
                  isBelowInput
                  isDanger
                >
                  {t('Incorrect derivation path')}
                </Warning>
              )}
```

The above is a bit weird. This is very spread, but it is what it is.

Reply:
arg sorry, should have seen that.